app/views/fagus/categories/index.json.jbuilder

```diff
@@ -0,0 +1,4 @@
+json.array! @categories do |category|
+  json.id category.id
+  json.name category.name
```

Comment:
I believe you need to add in `category.reference` if you are planning on using jquery.